src/views/templates/utils/virtualMachineTemplatesCPUMemory.ts

```diff
@@ -0,0 +1,42 @@
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
```

Comment:
The logic in this file was implemented already and exists here: https://github.com/kubevirt-ui/kubevirt-plugin/blob/main/src/utils/resources/vm/utils/disk/size.ts

Reply:
I'm not sure if all the logic from this file could be replaceable by what already exists...

Reply:
Or maybe you meant logic in https://github.com/kubevirt-ui/kubevirt-plugin/pull/151/files#diff-6384a372b7cedbc72c535dc614e0c3ebcb1f8937c7f6638028d30082dcb300b1 ?